osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -236,6 +243,11 @@ private void updateDisplayModeDropdowns()
                 displayDropdown.Show();
             else
                 displayDropdown.Hide();
+
+            if (host.Window?.SafeAreaPadding.Value.Total != Vector2.Zero)
+                safeAreaConsiderationsCheckbox.Show();
+            else
+                safeAreaConsiderationsCheckbox.Hide();
```

Comment:
Slightly safer to update visibility based on value changes to `host.Window.SafeAreaPadding` in case it gets updated later than the display.